omod/src/main/java/org/openmrs/module/auditlogweb/web/controller/ViewAuditController.java

```diff
@@ -0,0 +1,144 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.auditlogweb.web.controller;
+
+import lombok.RequiredArgsConstructor;
+import org.hibernate.QueryException;
+import org.openmrs.module.auditlogweb.api.AuditService;
+import org.openmrs.module.auditlogweb.api.utils.EnversUtils;
+import org.openmrs.module.auditlogweb.web.dto.AuditFieldDiff;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.ModelMap;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.servlet.ModelAndView;
+import javax.servlet.http.HttpServletRequest;
+import java.util.ArrayList;
+import java.util.List;
+import java.lang.reflect.Field;
+import java.util.Objects;
+
+@Controller
+@RequestMapping(value = ViewAuditController.MODULE_PATH + "/viewAudit.form")
+@RequiredArgsConstructor
+public class ViewAuditController {
+
+    public static final String MODULE_PATH = "/module/auditlogweb";
```

Comment:
I thought you were moving this into a separate class? 

https://github.com/openmrs/openmrs-module-auditlogweb/pull/11#discussion_r2149750181